examples/README.md

```diff
@@ -0,0 +1,17 @@
+# GitHub Provider Examples
+
+This directory contains a set of examples of using various GitHub services with
+Terraform. The examples each have their own README containing more details
+on what the example does.
+
+To run any example, clone the repository and run `terraform apply` within
```

Comment:
could we also specify the `init` and `plan` commands in the example? might be useful to add the complete flow for newer TF practitioners..hmm and possibly note to set variable definitions in a `.tfvars` file and pass as an arg to `apply` as one option

Reply:
```suggestion
To run any example, clone the repository and run `terraform init`, `terraform plan`, and `terraform apply` within
```